go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.1 // indirect
 	github.com/kr/pretty v0.3.1 // indirect
 	github.com/nxadm/tail v1.4.4 // indirect
+	github.com/sanity-io/litter v1.5.5 // indirect
```

Comment:
this should be a submodule of e2e/ not agentbaker/?

Reply:
think this is the only real issue...

Reply:
nuked it